### PR TITLE
20210702 new features api

### PR DESCRIPTION
**Pull request by shivamag00**
Review Requested: @jrobbins 

In this PR, 

- implemented get method for the features api

### CODE REVIEW

File: pages/schedule.py [hunk 1]
```diff
@@ -97,11 +97,7 @@ class ScheduleHandler(basehandlers.FlaskHandler):
...
 
   def get_template_data(self):
-    user = users.get_current_user()
-    features = models.Feature.get_chronological(
-        show_unlisted=permissions.can_edit_any_feature(user))
     template_data = {
-      'features': json.dumps(features),
       'channels': json.dumps(construct_chrome_channels_details(),
                              indent=4)
     }
```
Comment (shivamag00) [inline pages/schedule.py:103]: The releases schedule fetches all features list using JS. The features defined in `template_data` is not used in the template `schedule.html`.

**Review by shivamag00 — COMMENTED** [review]
@jrobbins, please have a look at this

File: pages/schedule.py (hunk 1 above)
Comment (jrobbins) [reply pages/schedule.py:103]: Good point.

**Review by jrobbins — APPROVED** [review]

**Comment by shivamag00** [comment]
This resolved issue #1412. Hence we may close it.

**Pull request merged**